ydb/public/sdk/cpp/src/client/rate_limiter/rate_limiter.cpp

```diff
@@ -45,6 +96,105 @@ void THierarchicalDrrSettings<TDerived>::SerializeTo(Ydb::RateLimiter::Hierarchi
     if (PrefetchWatermark_) {
         proto.set_prefetch_watermark(*PrefetchWatermark_);
     }
+
+    if (ImmediatelyFillUpTo_) {
+        proto.set_immediately_fill_up_to(*ImmediatelyFillUpTo_);
+    }
+
+    if (LeafBehavior_) {
+        LeafBehavior_->SerializeTo(proto);
+    }
+}
+
+TMetric::TMetric(const Ydb::RateLimiter::MeteringConfig_Metric& proto) {
+    Enabled_ = proto.enabled();
+    if (proto.billing_period_sec()) {
+        BillingPeriod_ = std::chrono::seconds(proto.billing_period_sec());
+    }
+    for (const auto& [k, v] : proto.labels()) {
+        Labels_[k] = v;
+    }
+    if (proto.has_metric_fields()) {
+        TString jsonStr;
```

Comment:
```suggestion
        TStringType jsonStr;
```
В репах ydb и ydb-cpp-sdk Protobuf либы разные строки используют

Reply:
https://github.com/ydb-platform/ydb/pull/14428